common/IoT-Introduction.md

```diff
@@ -1,3 +1,121 @@
+
+All the connections between the three parts are wireless. Which wireless protocol do we choose for the applications? How do we achieve all functionalities like network create/join/leave, interaction with end devices through Hub/Gateway or mobile device app? We will talk more about it in the following sections. 
+
+# 3. Choose wireless protocol  
+The wireless protocols vary that include WiFi, Zigbee, Bluetooth Mesh, Z-Wave.etc. Every wireless protocol is designed for a purpose. 
+
+__WiFi__ is the most popular wireless protocol we are using in daily life. It is based on IEEE 802.11. It is for high-speed data transfer usage. It generally has a WiFi router to create the network. Each end device searches the SSID, joins the network with user name and password. That is how our computer and mobile device surging the internet via WiFi. 
+
+WiFi's high data rate is matched by high power consumption, making it a poor fit for battery power devices. And it is a star network. Each device interacts with the router, cannot directly talk with other end devices. Some end devices might have a problem to connect with the router at the corner of the network. That how we are seeking the WiFi signal in our daily life. WiFi can be used in smart speaker (Hub/Gateway) to connect with cloud service for interpreting natural voice, or accessing data and sending command from mobile device. 
+
+<div align="center">
+<img src="images/CM-IoT-Introduction/2019-12-17-17-47-55.png">
+</div>
+
+__Zigbee__ is very popular in IoT applications. It is based on the IEEE 802.15.4 standard defined in 2003. Zigbee enables mesh networking, low power consumption, lower data rate, allowing sensors and other end devices to connect to each other over longer distance.  Because of the protocol’s technological limitations about data rate, range, and power consumption, Zigbee is more appropriate for small-scale powered wireless networks.
+
+__Bluetooth Mesh__ is a mesh network released in July 2017. 
+
+<div align="center">
+<img src="images/CM-IoT-Introduction/2019-12-17-17-48-40.png">
+</div>
+
+ISM(Industrial Scientific Medical) Band 2.4G is reserved internationally for the use of radio frequency (RF) energy. It supports WiFi (IEEE 802.11b/g/n), Zigbee/Thread (IEEE 802.15.4), Bluetooth®, and Bluetooth low energy. 
+
+__WiFi__ contains 13 overlapping channels (1-13) with center frequencies from 2.412GHz to 2.472GHz. Plus the 14th channel at 2.484 GHz. it is designated in the 2.4 GHz range, spaced 5 MHz apart from each other except for a 12 MHz space before channel 14. Each channel has a nominal 22MHz bandwidth, but 802.11n can bond two adjacent channels resulting in a nominal 44MHz bandwidth.
+
+__ZigBee__ contains 16 channels (11-26) with center frequencies from 2.405GHz to 2.480GHz and each channel with a nominal 2MHz bandwidth. 
+
+__Bluetooth Mesh__ contains 40 channels with 2 MHz channel spacing. Three of the 40 channels are advertising channels (shown in red)
```

Comment:
Is it needed to mention that the Bluetooth mesh use only 3 advertising channels?
Bluetooth low energy (BLE) specification defines 40 channels with 2 MHz channel spacing. Three of the 40 channels are advertising channels (shown in red), used for device discovery, connection establishment, and broadcast. Bluetooth Mesh is a specification on top of Bluetooth 4.0 which use only 3 advertising channels.